src/dmd/console.d

```diff
@@ -13,13 +13,18 @@
 module dmd.console;
 
 import core.stdc.stdio;
-extern (C) int isatty(int) nothrow;
 
 version (Windows)
 {
     import core.sys.windows.winbase;
     import core.sys.windows.wincon;
     import core.sys.windows.windef;
+
+    version (CRuntime_DigitalMars)
+      private extern (C) int isatty(int) @trusted @nogc nothrow;
+    else version (CRuntime_Microsoft)
+      private extern (C) int _isatty(int) @trusted @nogc nothrow;
+    static assert(0, "Unsupported Windows runtime.");
```

Comment:
Missing else. But really, let's just continue on using the 'deprecated' Posix name for MSVC too and be done with this.

Reply:
Can you present a reason? I don't want to disagree, but I don't understand why going that way either. If someday Microsoft decides to remove it, this will break and they have a justification, because of being deprecated. If they want to remove `_isatty`, they first need to deprecate it. I don't know how is the deprecation process for Microsoft libraries tho.

I mean, we can detect it right away, but aren't old builds going to stop working or is the Microsoft runtime library statically linked? If yes, this is not going to be a big deal on bootstrapping on a different system.

Reply:
I don't really want to argue about this, but the extra lines don't buy us anything really. Microsoft deprecated these Posix names for as long as I can remember, and they provide the aliases in `oldnames.lib`, which is pulled in by both DMD and LDC by default, as druntime uses the Posix names in various places. Microsoft ditching that oldnames.lib, with all the compatibility legacy they've got, is IMO extremely unlikely. Should it ever happen, we can e.g. add some `pragma(mangle)` stuff and blow up the number of lines a bit.

Reply:
Ok. I kept it.